lib/node_modules/@stdlib/math/base/special/besselj0/examples/index.js

```diff
@@ -18,13 +18,14 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
 var besselj0 = require( './../lib' );
 
 var x;
-var i;
 
-for ( i = 0; i < 100; i++ ) {
-	x = randu() * 100.0;
-	console.log( 'besselj0(%d) = %d', x, besselj0( x ) );
-}
+x = uniform( 100, 0.0, 100.0, {
```

Comment:
You need to declare and assign in these files, as well.

```suggestion
var x = uniform( 100, 0.0, 100.0, {
```